e2e/process-services/tasks/custom-tasks-filters.e2e.ts

```diff
@@ -64,8 +64,8 @@ describe('Start Task - Custom App', () => {
     const totalNrOfPages = 'of 4';
     const currentDateStandardFormat = DateUtil.formatDate('YYYY-MM-DDTHH:mm:ss.SSSZ');
     const currentDate = DateUtil.formatDate('MM/DD/YYYY');
-    const beforeDate = moment().add(-1, 'days').format('MM/DD/YYYY');
-    const afterDate = moment().add(1, 'days').format('MM/DD/YYYY');
+    const beforeDate = format(addDays(new Date(),-1), 'MM/dd/yyyy');
```

Comment:
Suggestion: you can use subDays()  to subtract days instead of adding -1.

Reply:
Review comments resolved